Firestore/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 - [changed] Add more details to the assertion failure in Query::Comparator() to
   help with future debugging (#9258).
+- [fixed] Fixed an issue where returning `nil` from the update closure when
```

Comment:
We should probably elude to the fact that this might be a breaking change that can be addressed by importing FirebaseFirestoreSwift.

Reply:
Done.